pkgs/top-level/python-packages.nix

```diff
@@ -14941,6 +14941,72 @@ in {
     };
   };
 
+  pythonnet = buildPythonPackage rec {
+    name = "pythonnet-${version}";
```

Comment:
`name` is generated automatically. `pname` should be provided instead

Reply:
Done